docs/config/README.md

```diff
@@ -50,6 +50,13 @@ minio server ~/Photos
 |:---|:---|:---|
 |``region``| _string_ | `region` describes the physical location of the server. By default it is set to `us-east-1`, which is same as AWS3's default region. If you are unsure leave it to default.|
 
```

Comment:
Since we decided to document this.. we should add it with `minio server` 

```
$ minio server <command help>
ENVIRONMENT VARIABLES:
  ACCESS:
     MINIO_ACCESS_KEY: Custom username or access key of 5 to 20 characters in length.
     MINIO_SECRET_KEY: Custom password or secret key of 8 to 40 characters in length.

  BROWSER:
     MINIO_BROWSER: To disable web browser access, set this value to "off".
```